spec/report_spec.rb

```diff
@@ -58,6 +58,46 @@
       expect(report).to be_an_instance_of(described_class)
       expect(report.id).to match('trkrep')
     end
+
+    it 'creates a report with custom columns and additional columns' do
+      report = described_class.create(
+        start_date: Fixture.report_start_date,
+        end_date: Fixture.report_end_date,
+        type: 'shipment',
+        columns: ['usps_zone'],
+        additional_columns: %w[from_name from_company],
+      )
+
+      expect(report).to be_an_instance_of(described_class)
+      expect(report.id).to match('shprep')
```

Comment:
Nit: Could do without the ID match here since the primary objective of the test is to make sure it's a report with the columns requested.

Reply:
Resolved in 838f54d0dbb44f5b78595d4a0840983b7d997834